<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>foxTextClass</class>
 <widget class="QWidget" name="foxTextClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>foxText</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {background-color: rgb(255, 255, 255);border:none}
QPushButton:hover {background-color: rgb(103, 209, 255);}
QComboBox {background-color: rgb(255, 255, 255);border:none}
QComboBox:hover {background-color: rgb(103, 209, 255);}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>配置元素类</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="foxText.qrc"/>
 </resources>
 <connections/>
</ui>
